From ef60c96d9c6b2972556c08f7c0763f033372d9ba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Mon, 3 Apr 2023 15:52:46 +0300
Subject: [PATCH 16/28] [libcxxabi] [test] Mark
 thread_local_destruction_order.pass.cpp unsupported for mingw targets

This can be fixed in a future mingw-w64 crt update; the issue is
that TLS destructors that are registered while executing TLS
destructors are lost and leaked.
---
 libcxxabi/test/thread_local_destruction_order.pass.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libcxxabi/test/thread_local_destruction_order.pass.cpp b/libcxxabi/test/thread_local_destruction_order.pass.cpp
index 9daf966c209c..49fc8a38be33 100644
--- a/libcxxabi/test/thread_local_destruction_order.pass.cpp
+++ b/libcxxabi/test/thread_local_destruction_order.pass.cpp
@@ -10,6 +10,7 @@
 // UNSUPPORTED: no-threads
 
 // XFAIL: LIBCXX-FREEBSD-FIXME
+// UNSUPPORTED: target={{.*-windows-gnu}}
 
 #include <cassert>
 #include <thread>
-- 
2.34.1

